hoomd_organics/base/simulation.py

```diff
@@ -531,7 +725,42 @@ def run_NPT(
         thermalize_particles=True,
         write_at_start=True,
     ):
-        """"""
+        """
+        Run the simulation in NPT dynamics.
```

Comment:
I think it would be more correct to say "Run the simulation in the NPT ensemble". The dynamics thing really only applies to the Langevin method